ingest-app/src/cmr/ingest/validation/validation.clj

```diff
@@ -88,9 +88,13 @@
   [concept]
   (if-errors-throw :bad-request
                    (if (mt/umm-json? (:format concept))
-                     (umm-spec/validate-metadata (:concept-type concept)
-                                                 (:format concept)
-                                                 (:metadata concept))
+                     (let [umm-version (mt/version-of (:format concept))
+                           accept-version (config/ingest-accept-umm-version)]
+                       (if (>= 0 (compare umm-version accept-version))
+                         (umm-spec/validate-metadata (:concept-type concept)
+                                                     (:format concept)
+                                                     (:metadata concept))
+                         [(str "umm version is higher than ingest-accept-umm-version" (pr-str accept-version))]))
```

Comment:
This needs a more user friendly message. Something like "UMM version X is in development. Can only ingest version 1.8 or lower." replace 1.8 with accept-version. This message would only work though if we validated that the version they're ingesting is legitimate first (see comment below). Otherwise it would have to be something different.